File: 4005-maximum-total-subarray-value-i/maximum-total-subarray-value-i.cpp
#define ll long long int

class Solution{
public:
    long long maxTotalValue(vector<int> &nums, int k){
        int n=nums.size();

        ll mx=*max_element(nums.begin(), nums.end());
        ll mn=*min_element(nums.begin(), nums.end());

        // cout << mx << " " << mn << endl;

        return (mx-mn)*(ll)k;


    }
};

File: 4005-maximum-total-subarray-value-i/maximum-total-subarray-value-i.java
import java.util.*;

class Solution{
    public long maxTotalValue(int[] nums, int k){
        // long mx=Arrays.stream(nums).max().getAsInt();
        // long mn=Arrays.stream(nums).min().getAsInt();

        // return (mx-mn)*(long)k;

        IntSummaryStatistics stats = Arrays.stream(nums)
                                           .summaryStatistics();
        return (stats.getMax()-stats.getMin())*(long)k;
    }
}